.gitmodules

```diff
@@ -13,3 +13,7 @@
 [submodule "crates/wasi-crypto/spec"]
 	path = crates/wasi-crypto/spec
 	url = https://github.com/WebAssembly/wasi-crypto.git
+[submodule "crates/fuzzing/wasm-spec-interpreter/ocaml/spec"]
+	path = crates/fuzzing/wasm-spec-interpreter/ocaml/spec
+	url = https://github.com/WebAssembly/spec
+	shallow = true
```

Comment:
This doesn't help. Cargo (using libgit2) does a deep clone of the spec and then recursively clones all submodules of the spec. That is `KaTeX/KaTeX`, `KaTeX/katex-fonts` and `KaTeX/katex-test-fonts` for a total checkout size of 260MB! That is reaching the size of the LLVM submodule of rust, which is a real pain to download and for that reason has even be made to only lazily get cloned when necessary. It is not necessary for `./x.py check`, but this submodule will get recursively cloned with `./x.py check` by cargo when building cg_clif.

Could you please instead vendor just the interpreter? The interpreter itself is less than 400kb.